File: blog/apps/post/urls.js
$L('doff.conf.urls', '*');
$L('blog.apps.post.views');

var urlpatterns = patterns('',
    ['^add_tag/$', 'blog.apps.post.views.add_tag'],
    ['^remove_tag/([A-Za-z0-9-]+)/$', 'blog.apps.post.views.remove_tag'],
    ['^add_post/$', views.add_post],
    ['^remove_post/([A-Za-z0-9-]+)/$', views.remove_post]
);

$P({ 'urlpatterns': urlpatterns })

File: sources/protopy/demo/urls.py
# -*- coding: utf-8 -*-


from django.conf.urls.defaults import *
from blog.post import views
    
urlpatterns = patterns('django.views.generic.simple', 
    (r'^add_tag/$', views.add_tag),
    (r'^remove_tag/(?P<slug>\w+)/$', views.remove_tag),
    (r'^add_post/$', views.add_post),
    (r'^remove_post/(?P<slug>\w+)/$', views.remove_post)
)